tracker_doctor.go

```
@@ -0,0 +1,838 @@
+		Name:   "Dippin Language",
+		Status: "ok",
+		Details: []CheckDetail{{
+			Status:  "ok",
+			Message: fmt.Sprintf("dippin %s at %s", ver, path),
+		}},
+		Message: fmt.Sprintf("dippin %s", ver),
+	}
+}
+
+func getDippinVersion(path string) string {
+	out, err := exec.Command(path, "--version").CombinedOutput()
```

golangci-lint (2.11.4)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)